src/CUSTOMIZATION/translations/de.js

```diff
@@ -0,0 +1,282 @@
+    loginHint: 'Startet Anmeldevorgang und leitet den QRCODE-Scanner um.',
+
+    questionnaire: {
+      middleButtonFinished: 'Nächste Frage.Frage wird beantwortet.',
+      middleButtonUnfinished:
+        'Nächste Frage, obwohl die Frage noch nicht beantwortet wird.',
+      middleButtonHint:
+        'Springt zu den nächsten Fragen oder schließt das Modal, wenn dies die letzte Frage war.',
+
+      rightButtonHint: 'Schließt das Modal.',
+      leftButtonHint: 'Springt zur vorherigen Frage.',
+      alertButtonHint: 'Schließt den Alarm.',
+
+      questionnaireCellHint:
+        'Öffnet neue Seite, um die Kategorien der Fragebogen anzuzeigen.',
+      categoryCellHint:
+        'Öffnet MODAL, um diesen Fragebogeneintrag auszufüllen.',
+      category: 'Diese Kategorie',
+      questionnaire: 'Dieser Fragebogen',
+      notStarted: 'wurde noch nicht beantwortet.',
+      notFinished:
+        'wurde bereits gestartet, ist aber noch nicht vollständig ausgefüllt.',
+      finished: 'ist vollständig vollständig.',
+
+      sendHint: 'Sendet den Fragebogen an die Bewertung.',
+
+      multipleChoice:
+        'Multiple Choice - mehr als eine Antwortoption kann ausgewählt werden.',
+      singleChoice:
+        'Einzelne Wahl - Nur eine Antwort-Option kann ausgewählt werden.',
+      textFieldHint:
+        'Bitte füllen Sie hier die angeforderten Informationen aus.',
+      dateFieldHint: 'Öffnet ein neues Modal, um ein Datum auszuwählen.',
+      sliderFieldEquals: 'entspricht',
+      sliderFieldAnd: ', und',
+
+      triggerHint:
+        'Wenn dieses Element ausgewählt ist, wird eine weitere zusätzliche Frage angezeigt, die mit diesem Element verbunden ist.Bitte füllen Sie auch die zusätzliche Frage aus.',
+    },
+  },
+
+  /** contains all string rendered on the login-screen, as well as the ones for the landing-screen */
+  login: {
+    title: 'Anmeldung',
+    subTitle: ' ',
+    submit: 'Anmeldung',
+    user: 'Subjekt-ID.',
+    pass: 'Passwort',
+    waiting: 'Einloggen...',
+    errorPass: 'Bitte versuche es erneut.',
+    noSubjectId: 'Kein gültiges SubjektD gefunden',
+    errorUserUnauthorized: 'Ungültige Benutzerdaten.',
+    qrInfo: 'Bitte zeigen Sie die Kamera auf den QR-Code.',
+    errorUserGeneric: 'Beim Login trat ein Problem auf.',
+    permissionDialog:
+      'Bitte erlauben Sie der App, auf Ihre Kamera zuzugreifen.',
+    nextStepAfterError: 'Bitte versuchen Sie es später erneut anzumelden.',
+
+    /** contains all strings of the landing-screen */
+    landing: {
+      buttonText: 'Navigieren Sie zum Anmeldebildschirm',
+      title: ' ',
+      subTitle: ' ',
+      welcomeTitle: 'Willkommen beim Compass \n  (Native Client reagieren)',
+      text: 'Dies ist der Lande-Bildschirm. \n  \n it kann verwendet werden, um eine Begrüßungsnachricht anzuzeigen, detailliert den folgenden Login-Prozess zu detaillierenAktualisiert durch Aktualisierung des Ordners Anpassen.',
```

Comment:
Typo: Er instead of it
What do you mean with "detaillierenAktualisiert". It doesn't make sense.